src/nvmeof/NVMeofGwMonitorClient.cc

```diff
@@ -298,7 +298,6 @@ void NVMeofGwMonitorClient::shutdown()
     std::lock_guard bl(beacon_lock);
     timer.shutdown();
   }
-
```

Comment:
This commit now removes a single empty line and does nothing else.  Were the other changes already merged?